tests/test_analysis.py

```diff
@@ -0,0 +1,95 @@
+import covasim as cv
+import numpy as np
+
+
+# Suite of tests to test basic functionality of the analysis.py file
+# Runtime: 5.004 seconds
+def test_analysis_snapshot():
+    sim = cv.Sim(analyzers=cv.snapshot('2020-04-04', '2020-04-14'))

```

Comment:
Suggest adding variable like `default_pop_size = 1000` to the top and then change each invocation of `sim` to e.g. `sim = cv.Sim(pop_size = default_pop_size, ...)`. This should reduce runtime from ~5 s to <1 s.